bundledata_test.go

```diff
@@ -570,6 +592,116 @@ func (s *bundleDataSuite) TestVerifyBundleWithRelationNameBindingSuccess(c *gc.C
 	c.Assert(err, gc.IsNil)
 }
 
+func (s *bundleDataSuite) TestParseKubernetesBundleType(c *gc.C) {
+	data := `
+bundle: "kubernetes"
+
+applications:
+    mariadb:
+        charm: "cs:mariadb-k8s-10"
+        scale: 2
+        placement: foo=bar
+    gitlab:
+        charm: "cs:gitlab-k8s-10"
+        num_units: 3
+        to: [foo=baz]
+    redis:
+        charm: "cs:redis-k8s-10"
+        scale: 3
```

Comment:
I do think that setting both of them should be an error. Both in the scale and placement.

Reply:
I considered it but thought it be best to be lenient on what we read in which is what one would normally do. I can error if you feel strongly about it.

Reply:
Ok, it will be an error to specify both. It does seem cleaner that way